### DISCUSSION TITLE
Flutter web client not working

### DISCUSSION THREAD

**Discussion by JelleBuning** [rustdesk/rustdesk#3742] [Q&A]
I am attempting to build the Flutter web client by following the build instructions shown in https://rustdesk.com/docs/en/dev/build/web/. Although i have managed to work around the first few errors, i can't seem to run the following command: "flutter run -d chrome". 

Running the command results in the following error:
![image](https://user-images.githubusercontent.com/37436672/226935297-86cf75ae-c599-4e30-8af2-dd1f92289a46.png)

Is the web build documentation still up to date?

**Comment by TBG-FR** [comment]
Did you manage to rebuild the flutter web client using a recent version of the repo and not the 2 years old one ? More specifically, did you manage to avoid that `dart:ffi` issue ?

**Reply by JelleBuning** [reply]
No unfortunately not, you'll have to use the version mentioned in the documentation.

**Reply by TBG-FR** [reply]
I'm facing a lot of issues with that version sadly... I'll keep trying, thanks for the answer !

**Reply by JelleBuning** [reply]
Maybe try the docker image under "how to build with docker"?

**Reply by TBG-FR** [reply]
Yes, or maybe starting back from the Android instructions but it's not easy to evaluate how complicated it'll be to port from Android to Web... 

Another solution could be to replace `dart:ffi` with https://github.com/EP-u-NW/web_ffi but it looks outdated

**Comment by rustdesk** [comment]
Update about web client: https://twitter.com/rustdesk/status/1771454382426448375
